ckan/templates/admin/snippets/confirm_delete.html

```diff
@@ -0,0 +1,28 @@
+{% extends "page.html" %}
+
+{% block subtitle %}{{ _("Confirm Delete") }}{% endblock %}
+
+{% block maintag %}<div class="row" role="main">{% endblock %}
+
+{% block main_content %}
+  <section class="module col-md-6 col-md-offset-3">
+    <div class="module-content">
+      {% block form %}
+      <p>
+        {% if ent_type == 'all' %}
+          {{ _('Are you sure you want to purge everything?') }}
+        {% else %}
+          {% set message = 'Are you sure you want to purge {ent_type}s?'.format(ent_type=ent_type) %}
```

Comment:
Don't rely on basic grammar rules and use `ungettext` for proper handling of plural form. Take a look at `search_result_text.html` - it's pretty similar to current template in terms of wording.

Reply:
I'm not entirely sure how to use ungettext here. The entity type in `message`  is always plural, but `ent_type` variable is in singular form for a logic purpose.

Reply:
If there is no difference between purging single entity and multiple entities, just use static messages without any substitution, like on lines 18, 24 and 30 - https://github.com/ckan/ckan/blob/master/ckan/templates/snippets/search_result_text.html